pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/Statistics.sql.stg

```diff
@@ -0,0 +1,87 @@
+group Statistics;
+
+getParticipantStatistics() ::= <<
+select
+    case when ug.user_governance_id is null then 'self' else 'governed' end as user_type,
```

Comment:
What happens if user does have a parent/proxy but not for the study (or it's not active)? I think we should check `usg.user_study_governance_id is null` instead.

Reply:
Changed